Readme.md

```diff
@@ -7,4 +7,8 @@ Current APIs:
 
 - S3
 
-Targeted for Java 1.6+
\ No newline at end of file
+Targeted for Java 1.6+
+
+**Currently, read timeout is disabled to avoid conflicts in updating.**
+
+for more information about timeouts, please see the following [wiki page](https://github.com/EMCECS/ecs-object-client-java/wiki/Example-for-avoiding-conflicts-when-timeout-is-enabled)
```

Comment:
I intended to use the existing "Changing timeouts" wiki page here: https://github.com/EMCECS/ecs-object-client-java/wiki/Changing-Timeouts - I added the section at the top explaining the general issue with client timeouts, but we would need to add a couple sections below and explain the defaults for different versions (versions between a.b.c and x.y.z have read timeout defaulted to 1 minute - all other versions should be defaulted to off), and show how you configure timeouts prior to this next release and after this next release.

I'm fine using either page, but let's just have one page and not duplicate information, and it needs to be comprehensive and include all of the information in one place.

Reply:
editd the existing "Changing timeouts"